src/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -80,20 +80,28 @@ public partial class HttpClientHandler : System.Net.Http.HttpMessageHandler
         public HttpClientHandler() { }
         public bool AllowAutoRedirect { get { throw null; } set { } }
         public System.Net.DecompressionMethods AutomaticDecompression { get { throw null; } set { } }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
```

Comment:
>Seems to me though that this attribute should exist in the reference assemblies for netstandard.dll, but not for .NET Core itself.

Ok. That makes sense if we want to somehow "hide" these new properties from netstandard20. But as you mentioned, we should NOT hide them for netcore itself.